testing/simapp/upgrades/upgrades.go

```diff
@@ -79,3 +80,18 @@ func CreateV7UpgradeHandler(
 		return mm.RunMigrations(ctx, configurator, vm)
 	}
 }
+
+// CreateV7LocalhostUpgradeHandler creates an upgrade handler for the ibc-go/v7.1 SimApp upgrade.
+func CreateV7LocalhostUpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+	clientKeeper clientkeeper.Keeper,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		// explicitly update the IBC 02-client params with the new default allowed clients
+		params := clienttypes.NewParams(clienttypes.DefaultAllowedClients...)
+		clientKeeper.SetParams(ctx, params)
```

Comment:
I'm okay with this for testing, but I think our migration docs should retrieve the existing params so we don't accidentally remove any client types not in our default allowed clients variable

Reply:
Good point, I can update this to just append the localhost client ID to the existing `params.AllowedClients` list. 
I'll wait for #3201 so I can use the new constant

Reply:
merged!